test/built-ins/Object/assign/ObjectOverride-sameproperty.js

```diff
@@ -0,0 +1,15 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*--
+description: Object properties are assigned to target in ascending index order,
+			 i.e. a later assignment to the same property overrides an earlier assignment.
+es6id:  19.1.2.1
+includes: [assert.js]
+--*/
+
+var target = {a: 1};
+var result = Object.assign(target,{a:2},{a:"c"});
+
+assert.sameValue(result.a, "c", "Object.assign() will override the property with the last value of this property in the arguments, so the value should be 'c'. But we got a \'" +result.a 
```

Comment:
[The `assert.sameValue` method prints actual and expected values in case of failure](https://github.com/tc39/test262/blob/36cbdcf8ec8cae0e494741034a6cd7f6d83f9325/harness/assert.js#L33), so the assertion messages in this patch can be simplified. Not strictly necessary, though, so @bterlson should probably weigh in before you change it.